src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write code here
+const header = document.querySelector('thead');
+const body = document.querySelector('tbody');
+const rows = body.rows;
+
+header.addEventListener('click', e => {
+  const headerEl = e.target.closest('th');
+  const cellIndex = e.target.cellIndex;
+
+  if (!headerEl) {
+    return;
+  }
+
+  const sorted = [...rows];
+
+  sorted.sort((rowA, rowB) => {
+    if (cellIndex === 2) {
+      return +rowA.cells[cellIndex].innerHTML
+      - (+rowB.cells[cellIndex].innerHTML);
+    }
+
+    if (cellIndex === 3) {
+      return +rowA.cells[cellIndex].innerHTML.slice(1).replace(',', '')
```

Comment:
move text formatting into the separate fucntion

Reply:
done, it is working but tests have been failed